Formula/erlang.rb

```diff
@@ -40,13 +40,13 @@ class Erlang < Formula
   fails_with :llvm
 
   resource "man" do
-    url "https://www.erlang.org/download/otp_doc_man_18.3.tar.gz"
-    sha256 "978be100e9016874921b3ad1a65ee46b7b6a1e597b8db2ec4b5ef436d4c9ecc2"
+    url "http://erlang.org/download/otp_doc_man_19.0.tar.gz"
```

Comment:
It doesn't look like they [interact much on that twitter account](https://twitter.com/erlang_org/with_replies). Maybe there is some alternate way of communicating with them? IRC? GitHub?